### PR TITLE
JENKINS-62589 Compatibility with tables to divs

### PR DESCRIPTION
**Pull request by timja**
https://github.com/jenkinsci/ownership-plugin/pull/82 but without the core upgrade

if you want to test it with tables to div you'll either need the core upgrade or to package the hpi and install it to a different Jenkins, there's compilation failures on new core to do with generic type signatures

@oleg-nenashev

### CODE REVIEW

File: src/main/resources/org/jenkinsci/plugins/ownership/config/DisplayOptions/config.jelly [hunk 1]
```diff
@@ -22,18 +22,18 @@
...
 -->
 <?jelly escape-by-default='true'?>
-<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" 
-         xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define"
+         xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form" xmlns:o="/lib/ownership">
   <f:advanced title="${%Ownership Display Options}">
     <f:entry title="" description="">
-        <table width="100%">
+        <o:blockWrapper>
```
Comment (fqueiruga) [inline src/main/resources/org/jenkinsci/plugins/ownership/config/DisplayOptions/config.jelly:29]: What's the difference between this and using `<f:block>`

Comment (timja) [reply src/main/resources/org/jenkinsci/plugins/ownership/config/DisplayOptions/config.jelly:29]: block uses a `tr` not a `table`

seems to be that in a number of places `tr` and `td` were wrapped in tags but the top level table was not

Comment (fqueiruga) [reply src/main/resources/org/jenkinsci/plugins/ownership/config/DisplayOptions/config.jelly:29]: I see. Would it make sense to introduce something like this on Jenkins side?

Comment (timja) [reply src/main/resources/org/jenkinsci/plugins/ownership/config/DisplayOptions/config.jelly:29]: not sure, would have been great if it was there years ago, but may not be worth it now

**Pull request merged**